dynamic_programming/longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,56 @@
+"""
+Longest Palindromic Subsequence Problem:
+Find the longest sequence of characters from a given input string that
+is the same backwards and forwards
+https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+"""
+
+
+def longest_palindromic_subsequence(s: str, length: int) -> int:
```

Comment:
```suggestion
def longest_palindromic_subsequence(s: str, length: int) -> str:
```
We should return the sequence, not the length of the sequence.  The function is not named `length_of_the_longest_palindromic_subsequence()` so I would not expect an int result.

The caller can always call `len(result)` if they want the length.